ci/test/00_setup_env_win64.sh

```diff
@@ -14,3 +14,4 @@ export RUN_FUNCTIONAL_TESTS=false
 export RUN_SECURITY_TESTS="true"
 export GOAL="deploy"
 export BITCOIN_CONFIG="--enable-reduce-exports --disable-gui-tests --without-boost-process"
+export NO_WERROR=1
```

Comment:
I haven't tried this one yet. What is the error here?

Reply:
https://cirrus-ci.com/task/6268532532969472
```
script/interpreter.cpp: In function ‘bool EvalChecksig(const valtype&, const valtype&, prevector<28, unsigned char>::const_iterator, prevector<28, unsigned char>::const_iterator, ScriptExecutionData&, unsigned int, const BaseSignatureChecker&, SigVersion, ScriptError*, bool&)’:
script/interpreter.cpp:429:1: error: control reaches end of non-void function [-Werror=return-type]
 }
 ^
cc1plus: some warnings being treated as errors
```

Reply:
Thanks. Can confirm locally:

![Screenshot from 2020-12-03 18-29-59](https://user-images.githubusercontent.com/6399679/101065576-9a4bf300-3595-11eb-9a32-ea1d10c51c11.png)

Reply:
Could mention with a comment?